utilities/test_suite/HIP/runTests.py

```diff
@@ -51,33 +51,28 @@ def get_log_file_list(preserveOutput):
     ]
 
 def run_unit_test(srcPath1, srcPath2, dstPathTemp, case, numRuns, testType, layout, qaMode, decoderType, batchSize, roiList):
-    print("\n\n\n\n")
-    print("--------------------------------")
-    print("Running a New Functionality...")
-    print("--------------------------------")
+    print("\n")
```

Comment:
please remove this and combine with the next statement. I think it is better to use std::cout for all prints instead of print statements

Reply:
Hi Rajy,
The \n creates a separation between the build log and the current command. Removing it and combining with the next statement could add extra blank lines, especially with bitdepth and layout loops. The current approach keeps it to just one empty line at the start.
As this is a Python script, we used print() statements.